core/src/main/scala/kafka/admin/ReassignPartitionsCommand.scala

```diff
@@ -82,7 +82,7 @@ object ReassignPartitionsCommand extends Logging {
     }
   }
 
-  def generateAssignment(zkUtils: ZkUtils, opts: ReassignPartitionsCommandOptions) {
+  def generateAssignment(zkUtils: ZkUtils, opts: ReassignPartitionsCommandOptions): Map[TopicAndPartition, Seq[Int]] = {
```

Comment:
That's fine too. The main aim is to keep the method that returns a value without `println` so that it can be tested without polluting the logs, for example.